apps/src/music/blockly/FieldChord.ts

```diff
@@ -12,6 +12,7 @@ const experiments = require('@cdo/apps/util/experiments');
 const MAX_DISPLAY_NOTES = 3;
 const FIELD_WIDTH = 51;
 const FIELD_HEIGHT = 18;
+const FIELD_PADDING = 2;
```

Comment:
Should we share this constant between FieldChord and FieldPattern?

Reply:
Similarly here.